### ISSUE TITLE
feat(slam): HST run-variant folders and the Delaunay-1250 A100 legs

### ISSUE THREAD

**Issue by Jammy2211** [PyAutoLabs/autolens_inference#5]
## Overview

The first four A100 legs of the SLaM base run finished — dense and sparse, seeds 0 and 1, all five stages `completed` (~2570 s/seed dense, ~4012 s/seed sparse) — and are sitting untracked in `results/slam/imaging/hst/`. Every leg of every experiment on this cell lands directly in `results/slam/imaging/hst/<config_name>/`, and `build_readme.py` groups rows into a parity row by the payload's `target` (`hst/slam5/seed<n>`), so a second experiment on the same cell would share both the directory and the parity group — and claim to be the same run under a different backend.

This task gives the cell a **run-variant** level (`imaging/hst/slam_base/` for the mesh-28x28 workspace-default run), then adds the next experiment on it: a 1250-vertex **Delaunay** source on the A100, dense and sparse, seeds 0-1.

## Plan

- Add `--mesh {rect,delaunay}` and `--mesh-pixels N` to the inference CLI, and a run-variant segment (`slam_base` / `delaunay_1250`) to the results path, the output `path_prefix`, the target id and the result payload.
- Build the Delaunay stages with `al.mesh.Delaunay(pixels=1250, zeroed_pixels=0)` and `al.reg.AdaptSplit` — `al.reg.Adapt`, what the rect legs use, is not jit-traceable on the Delaunay family.
- Add a `scripts/imaging/slam/hst_delaunay.py` leaf so the wall-gate cell id is distinct from the rectangular cell's; a 1250-vertex Delaunay chain must never share a rate row with a 784-cell rectangular one.
- Move the four completed base rows under `slam_base/` and commit them — the first real results this repo holds — and teach `build_readme.py` the variant column and to skip rows that are not `complete`.
- Add the two A100 submits (`--array=0-1`, 12 h containment, `source: unmeasured probe-first: yes`) and file the Cortex task `slam_hst_delaunay_1250`.

<details>
<summary>Detailed implementation plan</summary>

### Affected Repositories
- autolens_inference (primary)

### Branch Survey
| Repository | Current Branch | Dirty? |
|-----------|---------------|--------|
| ./autolens_inference | main | untracked `results/slam/imaging/`, `results/simulators/` (the pulled A100 rows) |

`worktree_check_conflict slam-hst-variant-folders-delaunay-1250 autolens_inference` → no conflict.

**Suggested branch:** `feature/slam-hst-variant-folders-delaunay-1250`

### Implementation Steps

1. `_inference_cli.py` — add `--mesh {rect,delaunay}` (default `rect`) and `--mesh-pixels N` (default 28 → 28x28 rect shape; 1250 for delaunay); add `mesh` / `mesh_pixels` / `variant` to `InferenceCLI` and a `variant_name(cli)` helper → `slam_base` for rect-28, `delaunay_<N>` for delaunay. `--rect-mesh`, `--regularization`, `--inversion` semantics unchanged. Reuse the existing `delaunay_regularization()` helper (already resolves `AdaptSplit` with the production coefficients inner 0.1 / outer 10.0 / signal_scale 0.1).
2. `scripts/misc/slam/_runner.py` —
   - `results_paths()` → `results/slam/<dataset_class>/<instrument>/<variant>/<config_name>/stages_seed<n>.json`
   - `path_prefix` → `slam/<dataset_class>/<instrument>/<variant>/<config_name>/seed_<n>/`
   - target id: base stays `hst/slam5/seed<n>`; delaunay is `hst/slam5_delaunay1250/seed<n>` — a different experiment must never land in the base run's parity group.
   - mesh construction: `rect` unchanged; `delaunay` → `af.Model(al.mesh.Delaunay, pixels=<N>, zeroed_pixels=0)` at `source_pix[1]` and `source_pix[2]` with `AdaptSplit` in place of `al.reg.Adapt`. `light[1]` / `mass_total[1]` inherit through `source_result_for_source=source_pix_result_2`.
   - payload: add `variant`, `mesh`, `mesh_pixels`, resolved `regularization`; `mesh_shape` null on a delaunay row; `SCHEMA_VERSION` 1 → 2, readers tolerate v1 (the four existing rows are not rewritten).
3. `scripts/imaging/slam/hst_delaunay.py` — new leaf (~30 lines, mirrors `hst.py`), cell id `imaging/slam/hst_delaunay`.
4. `scripts/misc/tooling/build_readme.py` — variant/mesh column; skip rows whose `status` is not `complete` (this also disarms the truncated `__rate_probe_342695` artefact).
5. Move `results/slam/imaging/hst/hpc_a100_jax_gpu_{dense,sparse}_fp64/` → `.../hst/slam_base/…` and commit the four rows; keep the rate-probe directory out of `results/`. Mirror the move on RAL for `output/` and `results/`.
6. `hpc/batch_gpu/submit_slam_delaunay1250_hst_jax_gpu_{dense,sparse}` — `--partition=gpu --gres=gpu:1 --cpus-per-task=4 --array=0-1 --time=12:00:00`; `--mem` 64gb dense / 96gb sparse (base measured ~13 GB and ~21.6 GB RSS at `source_pix[1]` with 784 rect pixels, scaled by 1250/784); `WALL-BASIS` row `cell: imaging/slam/hst_delaunay device: a100 precision: fp64`, `source: unmeasured probe-first: yes`.
7. Tests + docs — extend `scripts/misc/test/{test_slam_runner,test_slam_results,test_build_readme,test_check_submits}.py`; update `scripts/imaging/slam/README.md`, `AGENTS.md`, `hpc/README.md`, and add the `wiki/project/state.md` journal entry for the completed base legs and this restructure.
8. Cortex task `autolens_inference/slam_hst_delaunay_1250`, then submit both arrays on RAL (`hpc/sync submit --gpu <bare-name>`) and record the job ids. The session ends at submission — nothing waits on the queue.

### Key Files
- `_inference_cli.py` — flags, variant naming, the existing `delaunay_regularization()` / `rect_mesh_classes()` helpers
- `scripts/misc/slam/_runner.py` — paths, target id, mesh construction, payload
- `scripts/imaging/slam/hst_delaunay.py` — new leaf / new cell id
- `scripts/misc/tooling/build_readme.py` — variant column, `complete`-only rows
- `hpc/batch_gpu/submit_slam_delaunay1250_hst_jax_gpu_{dense,sparse}` — the A100 legs
- `wiki/project/state.md` — the journal entry

### Verification
- The four test modules above + `ruff`.
- `python3 scripts/misc/wall/check_submits.py` GREEN on both new submits.
- `python3 scripts/misc/tooling/build_readme.py --check` clean, base rows rendering at the new path, no probe row.
- A `PYAUTO_TEST_MODE` delaunay leg on the euclid cell (the HST cell does not fit on this laptop under JAX) — proves the Delaunay+AdaptSplit model builds, traces under jit and writes a v2 row at the new path, before anything reaches the queue.

### Out of scope
- The four CPU legs of the base run (still awaiting the human's OK).
- Filling `wall/rates.py` with the now-measurable A100 rates for `imaging/slam/hst` — belongs in its own prompt.
</details>

## Original Prompt
<details>
<summary>Click to expand starting prompt</summary>

```
WE began our timing runs in autolens_inference, and the first runs looks good, I would now like us to movre these into a folder from imaging/hst to imaging/hst/slam_base, and for us to perform delaunay runs with 1250 pixels which go in this folder on a100, both via normal route and sparse.
```

</details>

**Comment by Jammy2211** [comment]
**Shipped as PR #6; the A100 legs are queued.**

- `feature/slam-hst-variant-folders-delaunay-1250` — three commits: the variant grammar + Delaunay route, the submits + docs + journal, and a memory re-sizing from the smoke.
- **A100 arrays submitted 2026-09-15 from the feature branch on the RAL clone:** `343143` (dense) and `343145` (sparse), seeds 0-1 each, 24 h containment, 96 gb, 8 CPUs. `343143_0` started on `euclid-ral-gpu-2` immediately. The Cortex ledger (`projects/autolens_inference.md`) carries both as open runs.
- The four completed A100 **base** rows are committed under `results/slam/imaging/hst/slam_base/` — the first real results this repo holds — and the RAL `output/` and `results/` trees were moved to match.

**Three defects the pre-submission smoke caught**, each of which would have cost an A100 slot: the Delaunay mesh must be a plain instance (an `af.Model` leaves `areas_factor` free with no prior config); it does not place its own vertices (a Hilbert image mesh must supply them through `adapt_images`); and at `--cores 2` the fit deadlocks, because the mesh reaches qhull via a `jax.pure_callback` and a thread pool narrower than the callback depth hangs — hence 8 CPUs on these legs.

**Two things for the reader of the eventual comparison:**

1. `source_pix[2]` carries **3 free parameters against the rectangular branch's 5** — the rectangular image mesh leaves `weight_power`/`weight_floor` free, the Hilbert weights are fixed at the production 3.5/0.01. This is Delaunay-as-production-runs-it versus the rectangular default, not a controlled single-variable swap.
2. On the base row, dense and sparse agree at the same seed to 0.0-0.6σ while **seeds of the same leg disagree by up to 0.9σ and ~3 nats**. The `slam_hst_base` witness bands (2 nats, 0.2σ) are therefore tighter than Nautilus's own seed reproducibility on this cell. Any Delaunay-vs-base comparison has to be seed-paired, and the bands themselves probably need rewriting.

Merge is human (`/prm`). The runs need no session: they are pulled and judged on a later `/cortex` check-in.

**Comment by Jammy2211** [comment]
**Shipped.** PR #6 merged as `ee31deb` (lint green: ruff, `build_readme --check`, `check_submits --check`, pytest 55, the simulator scripts).

Four commits:
- `c646b41` — the run-variant grammar and the Delaunay mesh route (schema v2, `hst_delaunay` leaf, `build_readme` variant column + `complete`-only rows), plus the four completed A100 base rows.
- `40dac96` — the two A100 submits, the docs for the variant grammar, and the `wiki/project/state.md` journal entry.
- `db49c3e` — both legs re-sized to 96 gb after the smoke measured peak RSS the other way round (9.3 GB dense vs 6.5 GB sparse, inverting the base run's ordering).
- `0df0486` — the CI witness leg off `--cores 2`, the one configuration the Delaunay route is known to deadlock in.

The A100 arrays **343143** (dense) and **343145** (sparse), seeds 0-1, are running from this branch on the RAL clone; the Cortex ledger `projects/autolens_inference.md` carries them as open runs and they are judged on a later `/cortex` check-in, not here.

Follow-ups deliberately left open (not blockers): the four CPU legs of `slam_hst_base` still await the human's OK; `wall/rates.py` still carries no a100 row although the completed base rows can now supply one; and the `slam_hst_base` witness bands (2 nats, 0.2σ) are tighter than Nautilus's own seed scatter on this cell, which wants rewriting before the CPU legs run.
